src/components/PluggableComponent/index.jsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
There is not another walkaround to avoid to disable this?

Reply:
I don't know if useDeepCompareEffect give you error with the lint but it's better to put the disable inline next to the dependencies array